### PR TITLE
Change the '/data' endpoint to return 'Hello Richie!!'

### PR DESCRIPTION
**Pull request by Richie78321**
Change the PrintWriter to return "Hello Richie!!" instead of "Hello world!".

### CODE REVIEW

File: portfolio/src/main/webapp/css/master.css [hunk 1]
```diff
@@ -1,3 +1,8 @@
+#fact-container {
+  height: 75px;
+  overflow-y: auto !important;
```
Comment (billyeh) [inline portfolio/src/main/webapp/css/master.css:3]: Maybe a comment explaining why !important here is necessary would be good.

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -34,10 +34,22 @@ <h1 class="display-1 text-center my-auto">Richie Goulazian</h1>
...
       <hr class="mb-4" />
 
-      <h3>About me</h3>
-      <p><i>tldr;</i> I am a computer science student currently studying at the University of Pittsburgh.</p>
-      <p>I think my most defining characteristic is that I am a big fan of learning by doing. Over the past four years, I have been
-      learning computer science through personal projects that span a lot of different disciplines.</p>
+      <div class="row">
+        <div class="col-md-9">
+          <h3>About me</h3>
+          <p><i>tldr;</i> I am a computer science student currently studying at the University of Pittsburgh.</p>
```
Comment (billyeh) [inline portfolio/src/main/webapp/index.html:40]: nit: realized usually it's spelled tl;dr :)

Comment (Richie78321) [reply portfolio/src/main/webapp/index.html:40]: Oh wow. I guess I'm not a real denizen of the internet yet :'(. Will fix.

**Comment by Richie78321** [comment]
> Are these the right commits with this pull request? Changes seem unrelated to the PR title

Yeah good catch. I completely forgot that I had an active PR and was just trying to pass commits between my local machine and GCloud shell.

File: portfolio/src/main/webapp/css/master.css (hunk 1 above)
Comment (Richie78321) [reply portfolio/src/main/webapp/css/master.css:3]: Good point. !important isn't necessary here, and even though I realized that I accidentally did not save the file before staging! Will update.

**Comment by billyeh** [comment]
> > Are these the right commits with this pull request? Changes seem unrelated to the PR title
> 
> Yeah good catch. I completely forgot that I had an active PR and was just trying to pass commits between my local machine and GCloud shell.

Gotcha. Yeah if we could update the PR title to describe what was actually merged that'd help anyone looking through the history in the future :)

**Pull request merged**